### DISCUSSION TITLE
How to add menu to a Listbox Item in Combobox?

### DISCUSSION THREAD

**Discussion by upender7270** [adobe/react-spectrum#7047] [Q&A]
I am adding an action menu to each item in the combobox. But on click menu, the item is getting selected and popover is closing.
How can I add an action menu to the item?

![image](https://github.com/user-attachments/assets/dc3f9459-dbaa-469a-8c53-98ca5702404e)

**Comment by snowystinger** [comment]
Thanks for the discussion, this is not valid aria, so unfortunately you cannot do this.

Instead, you could make the search into a button that looks like a search field. When clicked, it opens a popover with a textfield at the top, which would overlap the trigger button. It would handle filtering a gridlist below it. The gridlist could then have the name and menu, be selectable,  but also allow for menu interactions.
